### PR TITLE
Add comments on NetInfo.js

### PR DESCRIPTION
**Pull request by alvinthen**
Further describe the methods available on NetInfo.

Question: How do I update the docs on the website?

### CODE REVIEW

**Comment by facebook-github-bot-1** [comment]
By analyzing the blame information on this pull request, we identified @bestander and @satya164 to be potential reviewers.

**Comment by facebook-github-bot-0** [comment]
@alvinthen updated the pull request.

**Comment by janicduplessis** [comment]
The docs on the website are generated from the code so the comments you added should show up on the website. You can test it locally with
```
cd website
npm install
npm start
```

File: Libraries/Network/NetInfo.js [hunk 1]
```diff
@@ -168,6 +168,9 @@ const _isConnectedSubscriptions = new Map();
  * ```
  */
 const NetInfo = {
+  /*
+   * Invokes the handler whenever the network info changes. The handler is passed with any of the connectivity types listed above.
```
Comment (janicduplessis) [inline Libraries/Network/NetInfo.js]: Wrap the comments at around 80 characters.

File: Libraries/Network/NetInfo.js [hunk 2]
```diff
@@ -196,11 +202,18 @@ const NetInfo = {
...
   },
 
+  /*
+   * Returns a promise that returns any of the connectivity types listed above.
+   */
   fetch(): Promise {
     return RCTNetInfo.getCurrentConnectivity().then(resp => resp.network_info);
   },
 
   isConnected: {
```
Comment (janicduplessis) [inline Libraries/Network/NetInfo.js]: Those won't show on the website as it doesn't extract methods from a nested object :( Still good to have the comments.

File: Libraries/Network/NetInfo.js [hunk 3]
```diff
@@ -168,6 +168,9 @@ const _isConnectedSubscriptions = new Map();
  * ```
  */
 const NetInfo = {
+  /*
```
Comment (janicduplessis) [inline Libraries/Network/NetInfo.js]: Use doc blocks
```
/**
 *
 */
```
Pretty much just add one `*` to each block first line.

File: Libraries/Network/NetInfo.js (hunk 1 above)
Comment (alvinthen) [inline Libraries/Network/NetInfo.js]: @janicduplessis from the contributing guide, it says

> Do not wrap lines at 80 characters - configure your editor to soft-wrap when editing documentation.

Did I understand incorrectly?

Comment (satya164) [inline Libraries/Network/NetInfo.js]: Can you change it to `Invokes the listener whenever network status changes. The listener receives one of the connectivity types listed above.`

File: Libraries/Network/NetInfo.js [hunk 4]
```diff
@@ -184,6 +187,9 @@ const NetInfo = {
     };
   },
 
+  /*
+   * Stop listening to network info changes.
```
Comment (satya164) [inline Libraries/Network/NetInfo.js]: Can you change it to `Removes the listener for network status changes`

File: Libraries/Network/NetInfo.js (hunk 2 above)
Comment (satya164) [inline Libraries/Network/NetInfo.js]: If it's not gonna show up on the website, we shouldn't add a comment here, as its obvious by looking at the method signature. We should update Flow types though.

File: Libraries/Network/NetInfo.js (hunk 1 above)
Comment (janicduplessis) [inline Libraries/Network/NetInfo.js]: This is for markdown files in the docs section. We  wrap doc blocks at ~80 for js files.

**Comment by alvinthen** [comment]
I have updated as per requested. I moved the documentation of the methods of `isConnected` to the object itself, this can help developer to understand without going through the source code

**Comment by facebook-github-bot-9** [comment]
@alvinthen updated the pull request.

File: Libraries/Network/NetInfo.js [hunk 5]
```diff
@@ -196,10 +203,19 @@ const NetInfo = {
     _subscriptions.delete(handler);
   },
 
+  /**
+   * Returns a promise that returns one of the connectivity types listed above.
```
Comment (janicduplessis) [inline Libraries/Network/NetInfo.js]: Let's change this to `Returns a promise that resolves with one ...`

File: Libraries/Network/NetInfo.js [hunk 6]
```diff
@@ -196,10 +203,19 @@ const NetInfo = {
...
+   */
   fetch(): Promise {
     return RCTNetInfo.getCurrentConnectivity().then(resp => resp.network_info);
   },
 
+  /**
+   * An object with similar methods above but the listener receives a boolean
```
Comment (janicduplessis) [inline Libraries/Network/NetInfo.js]: `An object with the same methods as above ...`

**Comment by janicduplessis** [comment]
Can you just do those small wording changes then this is good to go, thanks!

**Comment by facebook-github-bot-3** [comment]
@alvinthen updated the pull request.

**Comment by janicduplessis** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**